### PR TITLE
[DUOS-1333][risk=no] Populate institution for LC calls by user

### PR DESCRIPTION
**Pull request by rushtong**
## Addresses
Partially addresses https://broadworkbench.atlassian.net/browse/DUOS-1333 and https://broadworkbench.atlassian.net/browse/DUOS-1335

This does not complete either story but does facilitate better handling in the UI

## Changes
This update fetches the institution in addition to the LC when querying by user id. This will populate the user data for the /me and get user by id endpoints.

----
Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Review by raejohanek — APPROVED** [review]
tested locally with /me endpoint works as expected, might want to add documentation to api-docs for all affected endpoints

File: src/main/resources/assets/api-docs.yaml [hunk 1]
```diff
@@ -3181,9 +3181,6 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/LibraryCard'
-              example:
-                libraryCard:
-                  $ref: '#/components/schemas/LibraryCard'
```
Comment (rushtong) [inline src/main/resources/assets/api-docs.yaml:3186]: Moved to LibraryCard so all references can see the example.

File: src/main/resources/assets/api-docs.yaml [hunk 2]
```diff
@@ -3588,8 +3585,11 @@ paths:
           description: Server error.
   /api/researcher/{userId}:
     get:
+      deprecated: true
       summary: Return researcher properties.
-      description: Describes all the properties for a reseacher user.
+      description: |
+        DEPRECATED: Use /api/user/me or /api/user/{id}
+        Describes all the properties for a reseacher user.
```
Comment (rushtong) [inline src/main/resources/assets/api-docs.yaml:3592]: Unnecessary endpoint that was also displaying some LC information incorrectly.

File: src/main/resources/assets/api-docs.yaml [hunk 3]
```diff
@@ -4774,6 +4756,8 @@ components:
         userId:
           type: integer
           description: ID of associated user
+        institution:
+          $ref: '#/components/schemas/Institution'
```
Comment (rushtong) [inline src/main/resources/assets/api-docs.yaml:4760]: Now visible for endpoints that support it.

**Review by raejohanek — APPROVED** [review]
library card examples look good on swagger

**Comment by rushtong** [comment]
@raejohanek Feedback addressed. While going through the api docs, I did find some other issues so tried to do a little bit of cleanup along with marking as deprecated an API that we really don't need now that the user endpoints are returning more complete information.

**Pull request merged**